spec/support/spec_helper.moon

```diff
@@ -102,3 +102,21 @@ export pump_mainloop = ->
   count = 0
   while count < 100 and C.g_main_context_iteration(howl_main_ctx, false) != 0
     count += 1
+
+export within_activity = (activity_function, on_show) ->
+  command_line = howl.app.window.command_line
+  command_line.orig_show = command_line.show
+  command_line.show = spy.new =>
+    @orig_show!
+    on_show!
+
+  ok, error = dispatch.launch activity_function
+  if not ok
+    print error
+
+  command_line.show = command_line.orig_show
+
+export get_list_items = (column=1, widget_name='completion_list') ->
```

Comment:
Too generic a name I think, better to have a descriptive name here, even if it's long. The specs can alias it themselves for better readability if required.